js-oop/game/game.js

```diff
@@ -0,0 +1,21 @@
+function GameObject(imageUrl) {
+    this.imageUrl = imageUrl;
```

Comment:
There is no imageUrl field in GameObject, it must be image initialized with new Image()